src/controllers/update/update.review.beneficial.owner.other.ts

```diff
@@ -41,7 +41,8 @@ export const get = (req: Request, res: Response, next: NextFunction) => {
       templateName: UPDATE_REVIEW_BENEFICIAL_OWNER_OTHER_PAGE,
       ...dataToReview,
       ...principalAddress,
-      ...serviceAddress
+      ...serviceAddress,
+      entity_number: appData[EntityNumberKey],
```

Comment:
would have been enough to pass in a boolean to suppress display but ok

Reply:
Yeah, could have but this keeps some consistency with how we conditionally display other areas of the journey and we also already had access to OE number for the "add" views without passing any new flag, so limited the change to two controllers for review.